WcaOnRails/app/models/post.rb

```diff
@@ -2,11 +2,41 @@
 
 class Post < ApplicationRecord
   belongs_to :author, class_name: "User"
+  has_many :post_tags, autosave: true, dependent: :destroy
 
   validates :title, presence: true, uniqueness: true
   validates :body, presence: true
   validates :slug, presence: true, uniqueness: true
 
+  def reload
+    @tags = nil
+    super
+  end
+
+  def tags=(new_tags)
+    @tags = new_tags
+  end
+
+  def tags
+    @tags ||= post_tags.pluck(:tag).join(",")
+  end
+
+  def tags_array
+    tags.split(",")
+  end
+
+  before_validation do
+    tags_array = tags.split(",")
```

Comment:
Whoops! I actually meant to get rid of the method because it wasn't used anywhere else, but I forgot to actually delete it.